datafusion/core/src/physical_plan/hash_aggregate.rs

```diff
@@ -516,88 +619,7 @@ fn group_aggregate_batch(
                 })
         })?;
 
-    Ok(accumulators)
-}
-
-async fn compute_grouped_hash_aggregate(
```

Comment:
This logic is moved into a combination of `GroupedHashAggregateStream::new` and `GroupedHashAggregateStream::poll_next`